src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -115,6 +116,18 @@ public PreviewControl(NodeViewModel nodeViewModel)
             }
         }
 
+        private void UpdateMargin()
+        {
+            var nodeWidth = smallContentGrid.MinWidth;
+            var previewWidth = Math.Max(centralizedGrid.ActualWidth, nodeWidth);
+            var margin = (previewWidth - nodeWidth)/2;
+            Margin = new System.Windows.Thickness {Left = -margin };
```

Comment:
two things:
- I'm not sure if it applies, but we may need to take into account hdpi screens here.
- can you add a comment on the logic here.